ops/log.py

```diff
@@ -28,6 +28,17 @@
 class JujuLogHandler(logging.Handler):
     """A handler for sending logs and warnings to Juju via juju-log."""
 
+    DEFAULT_FORMATTING = None
+    VERBOSE_FORMATTING = '%(name)s:%(lineno)-5d %(message)s'
+
+    @classmethod
+    def set_formatting(cls, format: str | None = DEFAULT_FORMATTING):
+        """Adjust the log format output."""
+        root_logger = logging.getLogger()
+        for handler in root_logger.handlers:
+            if isinstance(handler, cls):
+                handler.setFormatter(logging.Formatter(format or cls.DEFAULT_FORMATTING))
```

Comment:
I'm a little confused here. DEAULT is None, so why `format or DEFAULT`?
Is this to escape user supplying an empty string?